packages/config/config/devices/0x027a/zen27.json

```diff
@@ -480,5 +480,11 @@
 			"maxValue": 99,
 			"defaultValue": 0
 		}
-	]
+	],
+	"metadata": {
+		"inclusion": "1. Initiate inclusion (pairing) in the app (or web interface).\n2. Finalize inclusion at the switch. Tap the upper paddle of the dimmer 3 times quickly. The LED indicators will blink to signal communication and remain on for 2 seconds to confirm inclusion",
+		"exclusion": "1. Bring your Z-Wave gateway (hub) close to the switch if possible\n2. Put the Z-Wave hub into exclusion mode \n3. Tap the lower paddle on the dimmer 3 times quickly\n4. Your hub will confirm exclusion and the device will disappear from your\ncontroller's device list",
```

Comment:
Uhhhh. Even if they have this in their manual, I don't think we should ever advertise moving the controller.
https://github.com/zwave-js/node-zwave-js/issues/1995

Reply:
ping @aderusha :)

Reply:
Oh hey, thanks for the ping :D  Fixed, along with some other slights changes to align the wording used.

Reply:
Don't forget the other two PRs ;)